### PR TITLE
Renaming

### PR DESCRIPTION
**Pull request by ayshptk**
[diff between `collection.ts` and `bundle.ts`](https://www.diffchecker.com/xBTyOPDK)

### CODE REVIEW

File: src/common/module-type.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ import { $enum } from "ts-enum-util";
  */
 export enum ModuleType {
   CURRENCY = 0,
-  COLLECTION = 1,
+  COLLECTON = 1,
```
Comment (nachoiacovino) [inline src/common/module-type.ts]: typo here

File: src/modules/app.ts [hunk 2]
```diff
@@ -233,7 +245,8 @@ export class AppModule extends Module<ProtocolControl> {
   ): Promise<ModuleMetadata[]> {
     const moduleTypesToGet = filterByModuleType || [
       ModuleType.NFT,
-      ModuleType.COLLECTION,
+      ModuleType.COLLECTON,
```
Comment (nachoiacovino) [inline src/modules/app.ts]: same here

**Review by nachoiacovino — COMMENTED** [review]
Are we also doing the renaming for

NFT -> NFTCollection
Currency -> Token

the same way?

Also, we briefly mentioned with @jakeloo to do the Market -> Marketplace renaming too, is that still happening?

File: src/common/module-type.ts (hunk 1 above)
Comment (ayshptk) [reply src/common/module-type.ts]: this is an outdated  commit, fixed already [here](https://github.com/nftlabs/nftlabs-sdk-ts/blob/1cff35348c3e177b65976c6f39ccb060632e9f9b/src/common/module-type.ts#L9)

File: src/modules/app.ts (hunk 2 above)
Comment (ayshptk) [reply src/modules/app.ts]: this is an outdated commit, removed it already [here](https://github.com/nftlabs/nftlabs-sdk-ts/blob/1cff35348c3e177b65976c6f39ccb060632e9f9b/src/modules/app.ts#L240)

**Comment by ayshptk** [comment]
@nachoiacovino yep if this is the right way, ill quickly rename the other ones as well. as for the other comments, they are outdated commits do no need to worry about it

File: src/core/index.ts [hunk 3]
```diff
@@ -337,11 +337,14 @@ export class ThirdwebSDK {
 
   /**
    *
-   * @param address - The contract address of the given Collection module.
-   * @returns The Collection Module.
+   * @param address - The contract address of the given Bundle module.
+   * @returns The Bundle Module.
```
Comment (jakeloo) [inline src/core/index.ts:341]: We should properly document deprecation so it shows up in the doc https://tsdoc.org/pages/tags/deprecated/

File: src/core/index.ts [hunk 4]
```diff
@@ -337,11 +337,14 @@ export class ThirdwebSDK {
...
    */
-  public getCollectionModule(address: string): CollectionModule {
-    return this.getOrCreateModule(address, CollectionModule);
+  public getCollectionModule(address: string): BundleModule {
```
Comment (jakeloo) [inline src/core/index.ts:343]: I think we should type alias this. Otherwise, it'd breaking changes for typescripts.

File: src/modules/bundle.ts [hunk 5]
```diff
@@ -14,6 +15,14 @@ import { MetadataURIOrObject } from "../core/types";
 /**
  * @beta
  */
+
+
+export interface BundleMetadata{
```
Comment (jakeloo) [inline src/modules/bundle.ts:20]: we should probably `yarn format` this

File: src/modules/bundle.ts [hunk 6]
```diff
@@ -122,10 +145,22 @@ export class CollectionModule extends ModuleWithRoles<NFTCollectionContract> {
     );
   }
 
-  public async isApproved(address: string, operator: string): Promise<boolean> {
+  public async isApproved(address: string, operator: string, tokenContract?: string, tokenId?: BigNumberish): Promise<boolean> {
```
Comment (jakeloo) [inline src/modules/bundle.ts:148]: I'm confused, what's this `tokenContract` about? 
This is public interfaces, if we want to do it for our internal convenience we should move them to private functions.

Comment (jakeloo) [reply src/modules/bundle.ts:148]: @ayshptk we should change this for sure.

**Pull request merged**